src/test/java/org/mockito/internal/stubbing/defaultanswers/ReturnsSmartNullsTest.java

```diff
@@ -51,8 +53,9 @@ public void should_return_an_object_that_allows_object_methods() throws Throwabl
 
         Foo smartNull = (Foo) answer.answer(invocationOf(Foo.class, "get"));
 
-        assertContains("SmartNull returned by", smartNull + "");
-        assertContains("foo.get()", smartNull + "");
+        assertThat(smartNull.toString())
```

Comment:
I wonder if a risk of NPE is real in those tests. With `smartNull + ""` the nicer error message would be displayed when `smartNull` is `null`.